mods/content/fantasy/datum/hnoll/language.dm

```diff
@@ -1,2 +1,22 @@
 /decl/language/hnoll
 	name = "Hnoll Tongue"
+	desc = "While every hnoll culture across the Nine Mothers and beyond has their own \
+	local dialect, the matriarches over the years have taken great care to cultivate and \
+	maintain a shared tongue used at the yearly gatherings and for inter-settlement trade. \
+	It's generally considered rather stuffy and archaic if spoken in the 'correct' manner."
+	speech_verb = "growls"
+	ask_verb = "chuffs"
+	exclaim_verb = "howls"
+	colour = "serpentid_lang"
+	key = "j"
+	flags = LANG_FLAG_WHITELISTED
```

Comment:
just making sure, this means it's only able to be selected by characters who get it from their species/culture/etc, right?